### ISSUE TITLE
Notifications: Welcome to 4.0 Notification Needs to Change

### ISSUE THREAD

**Issue by eaquigley** [IQSS/dataverse#2966]
Line 218 in the bundle says: "Welcome to Dataverse 4.0! Please take a look around, try everything out, and contact support@dataverse.org if you have questions or feedback." 

Need to update this to not say 4.0. Also, I'm guessing we should take out support@dataverse.org as that is quite specific to the Harvard Dataverse and not what other installations will want.

**Comment by posixeleni** [comment]
Hope I am speaking in the right place. I also wanted to add that the Signature should be editable as well since @shlake reported to me that she was not able to edit that part of the notification letters. It says "Dataverse Project"

**Comment by pdurbin** [comment]
> I'm guessing we should take out support@dataverse.org as that is quite specific to the Harvard Dataverse and not what other installations will want.

Yes, let's please use the system email instead.

Given that and Eleni's requirement, this issue needs to be assigned to a developer.

**Comment by eaquigley** [comment]
Passing to @mheppler to update it to be the link to the support form pop up instead of an email address.

**Comment by mheppler** [comment]
Reviewed with @eaquigley and have come up with new notification messages related to creating an account, both that display in the Notifications tab on the Account pg, and that get emailed to the user.

`Welcome to {dataverseServiceBean.findRootDataverse().name} Dataverse! Get started by adding or finding data. Have questions? Check out the <a href="{systemConfig.guidesBaseUrl}/{systemConfig.version}/user/index.html" title="{dataverseServiceBean.findRootDataverse().name} Dataverse User Guide" target="_blank">User Guide</a> or contact <a href="#" onclick="event.preventDefault();PF('contactForm').show();" title=”Contact {dataverseServiceBean.findRootDataverse().name} Dataverse Support”>Dataverse Support</a> for assistance.`

`Welcome to Dataverse! Get started by adding or finding data. Have questions? Check out the User Guide at {systemConfig.getGuidesBaseUrl()}/{systemConfig.getVersion()}/user/ or contact Dataverse Support for assistance.`

**Comment by mheppler** [comment]
Updated welcome/create account-related app/email notifications.

Pushed this code to the [2966-4.0notification](https://github.com/IQSS/dataverse/tree/2966-4.0notification) branch.

**Comment by mheppler** [comment]
Passing this to QA and submitting a pull request.

@posixeleni -- regarding your question, let's review this in more detail when you return to the office. If I am understanding it correctly, the `notification.email.closing` signature can be edited in `Bundle.properties`. If there are any other outstanding issues related to that which we need to address, we can open a new issue specific to those.

**Comment by mheppler** [comment]
| Page/Requirements  | Issue# | Commit# |
| ------------- | ------------- | ------------- |
| **Account Pg Notification, Create Account**  |
| Change notification msg to open Contact Dataverse Support popup, improve text in general  | #2966  | 4e44d104080f271a971fee116596ca06aedcac51, 421f821208af1198cc1e3fd18e9be230379d34cf |
| **Email Notification, Create Account**  |
| Change notification msg to not include '4.0' or 'support@dataverse.org', to improve text in general, and add dynamic (installation/version) link to User Guide  | #2966  | f35995a73abeca6e35ced4db4329bf4f8b72aba9, 4e44d104080f271a971fee116596ca06aedcac51 |

**Comment by kcondon** [comment]
I believe this pull request is being held up by other issues in the request so have not yet tested.

**Comment by pdurbin** [comment]
4e44d10 introduced the regression at #3036 so I'm probably going to revert it.

**Comment by pdurbin** [comment]
@kcondon as I mentioned I'm putting this issue in the 4.4 milestone since that's the milestone for this issue where I just left a detailed comment: https://github.com/IQSS/dataverse/issues/3036#issuecomment-212050816

**Comment by kcondon** [comment]
OK, looks good, closing.
